src/colinajose/service/GradesService.java

```diff
@@ -0,0 +1,114 @@
+package colinajose.service;
+
+import colinajose.model.people.Student;
+import colinajose.model.schoolEntities.Course;
+import colinajose.model.schoolEntities.Grade;
+import colinajose.model.schoolEntities.GradingScale;
+import colinajose.model.schoolEntities.Kardex;
+import colinajose.model.schoolEntities.Subject;
+import datastructures.arraylist.MyArrayList;
+import datastructures.circulardoublylinkedlist.MyCircularDoublyLinkedList;
+
+public class GradesService {
```

Comment:
I leave this to your consideration but since this service doesn't have attributes these methods can be static

Reply:
@JurgenPadilla fixed in next commit